infrastructure/local/local-http-signatures/README.md

```diff
@@ -0,0 +1,34 @@
+# Local HTTP Signatures
+
+This is an app that generates HTTP digests and signatures for Postman, using the [http-signature-utils](../../../packages/http-signature-utils/) package.
+
+## Local Development
+
+### Building
+
+From the monorepo root directory:
+
+```shell
+pnpm --filter local-http-signatures build
+```
+
+## Running the Postman signature app
```

Comment:
Maybe we can move the `postman-scripts` directly into `infrastructure/local` ?

Reply:
Where are they now?

Reply:
just inside the app itself: `infrastructure/local/local-http-signatures`

Reply:
I prefer the way you have it now